.sonarcloud.properties

```diff
@@ -0,0 +1,15 @@
+# Path to sources
```

Comment:
Is this file even necessary, if it's empty?

Reply:
The file itself is truly necessary - more can be found here https://sonarcloud.io/documentation/autoscan/#how-to-activate-the-feature

The file can be empty (delete all comments), but to me, it's more clear like this.